### ISSUE TITLE
error message should updated while following query sent to server

### ISSUE THREAD

**Issue by liangliu** [milvus-io/milvus#1022]
**Describe the bug**
error message should updated while following query sent to server

```
req_partition = {
            "partition_name": self.fa.pystr(),
            "partition_tag": self.fa.pystr(),
        }
        req_partition['partition_name'] = '汉子'
        res = requests.post(HOST + 'tables/%s/partitions' % req['table_name'], json.dumps(req_partition))
```
current return message
`[19:49:55-INFO-root]: {"message":"Invalid table name: \u6C49\u5B50. The first character of a table name must be an underscore or letter.","code":9} (test_http_tablepartition.py:675)`

**Steps/Code to reproduce behavior**
**Expected behavior**
**Environment details**
**Screenshots**
**Additional context**

**Comment by BossZou** [comment]
Thank you. I think the reason is partition is stored in database as table. So in project we check partition by invoking table methods. Now I plan to separate them.

**Comment by liangliu** [comment]
get it. you mean fix this issue in 0.7.0 release?

**Comment by liangliu** [comment]
verified
